<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewHouseDialog</class>
 <widget class="QDialog" name="NewHouseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>100</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="addressLabelLayout">
       <item>
        <widget class="QLabel" name="addressLabel">
         <property name="text">
          <string>&amp;Address</string>
         </property>
         <property name="buddy">
          <cstring>addressLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="addressIcon">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="sovetdoma.qrc">:/icons/exclamation-small-red.png</pixmap>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="addressLineEdit"/>
     </item>
     <item row="1" column="0">
     <layout class="QHBoxLayout" name="savingPathLabelLayout">
     <item>
      <widget class="QLabel" name="savingPathLabel">
       <property name="text">
        <string>Saving &amp;path</string>
       </property>
       <property name="buddy">
        <cstring>savingPathBrowse</cstring>
       </property>
      </widget>
     </item>
     <item>
     <widget class="QLabel" name="savingPathIcon">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="sovetdoma.qrc">:/icons/exclamation-small-red.png</pixmap>
      </property>
     </widget>
     </item>
     </layout>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="savingPathLayout">
       <item>
        <widget class="QLineEdit" name="savingPathLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="savingPathBrowse">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="sovetdoma.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewHouseDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewHouseDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
